sympy/stats/crv_types.py

```diff
@@ -2850,6 +2918,16 @@ def Trapezoidal(name, a, b, c, d):
 class TriangularDistribution(SingleContinuousDistribution):
     _argnames = ('a', 'b', 'c')
 
+    @property
+    def set(self):
+        return Interval(self.a, self.b)
+
+    @staticmethod
+    def check(a, b, c):
+        _value_check(b > a, "Parameter b > %s. b = %s"%(a, b))
+        _value_check(And(a <= c, c <= b),
```

Comment:
According to latest refactoring of `_value_check`, `And(a <= c, c <= b)` should be written as, `(a <= c, c <= b)`.

Reply:
Done. Thanks!